package.json

```diff
@@ -40,5 +40,14 @@
     "webpack": "^5.70.0",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.7.4"
+  },
+  "dependencies": {
+    "@babel/core": "^7.17.5",
```

Comment:
Adding babel is unnecessary when we are enabling jsx: "react" in tsconfig. In next PR I will remove babel.